settings.gradle

```
@@ -0,0 +1,3 @@
+rootProject.name = 'travel_planner'
+include 'module_planner'
```

```suggestion
include ':module_planner'
```
이렇게 하는 것과는 어떤 차이가 있는 것인가요?